docs/components/overrides/Editor/Editor.js

```diff
@@ -7,12 +7,12 @@ import 'codemirror/mode/jsx/jsx'
 // We’re explicitly specifying Webpack loaders here so we could skip specifying them in Webpack configuration.
 // That way we could avoid clashes between our loaders and user loaders.
 // eslint-disable-next-line import/no-unresolved, import/no-webpack-loader-syntax
```

Comment:
This, as well as other outstanding `// eslint-disable-*` lines can be removed.

Reply:
I'm leaving these here as this is in docs/overrides, which means it is just overriding a styleguidist component. So it should be left as is so that it stays as similar as possible to the original.